xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserProfileSheet.xml

```diff
@@ -89,7 +89,12 @@ $services.localization.render('platform.core.profile.category.profile.disabled')
     #end
     &lt;div class='userInfo'&gt;
     #if($xcontext.action == 'view' &amp;&amp; $hasEdit)
-      &lt;div class='editProfileCategory'&gt;&lt;a href="$doc.getURL('edit', 'editor=inline&amp;amp;category=profile')"&gt;&lt;span class='hidden'&gt;$services.localization.render('platform.core.profile.category.profile.edit')&lt;/span&gt;&lt;/a&gt;&lt;/div&gt;
+      &lt;div class='editProfileCategory'&gt;
+       &lt;a title="$escapetool.xml($services.localization.render('platform.core.profile.category.profile.edit'))" 
+        href="$doc.getURL('edit', 'editor=inline&amp;amp;category=profile')"&gt;
+        &lt;span class='sr-only'&gt;$services.localization.render('platform.core.profile.category.profile.edit')&lt;/span&gt;
```

Comment:
The text within the ``span`` element needs to be XML encoded also (like you did above).